notifications_utils/jinja_templates/email_template.jinja2

```diff
@@ -149,19 +149,14 @@
                 </td>
               {% endif %}
               {% if brand_name %}
-                {% if brand_logo %}
-                <td width="5">{{ spacer(5) }}</td>
-                {% endif %}
-                <td width="5">{{ spacer(5) }}</td>
-                <td width="100%" bgcolor="{{brand_colour}}" valign="middle" align="left">
+                <td width="100%" bgcolor="{{brand_colour}}" valign="middle" align="left" style="padding: 14px 0 14px 10px">
```

Comment:
Confirm that the left-hand spacing here should always be `10px`. That the previous code had different left-hand spacing values<sup>1</sup> for with and without an image is wrong.

1. `10px` with an image, `5px` without an image